Platform/AlderlakeBoardPkg/Library/ShellExtensionLib/CmdGpio.c

```diff
@@ -179,7 +179,7 @@ GpioPadcalc(
     IN UINT32 PinNumber
 )
 {
-  UINT32 GpioPad;
+  UINT32 GpioPad = 0;
```

Comment:
Following EDK2 coding style, set the initial value separately.

Reply:
Initialization is done separate line